docs/appendix/converting_from_20.rst

```diff
@@ -0,0 +1,173 @@
+###########################################################
+Changes and Considerations Converting from NVIDIA FLARE 2.0
+###########################################################
+There are several breaking changes:
+
+Provisioning Changes
+====================
+The project.yml file for NVIDIA FLARE 2.1 uses api_version 3, and it is not compatible with previous API versions because
+the Provision Tool has been enhanced to support the configurations of additional components. See the example :ref:`project_yml`
+that shows how these components are configured in the components section of StaticFileBuilder.
+
+Overseer
+--------
+The Overseer is a new system component that determines the hot FL server at any time.
+The name of the overseer must be unique and in the format of fully qualified domain names.  During
```

Comment:
Should we also discuss the option of using a unique hostname rather than FQDN, and mapping IP->host via /etc/hosts here?  We see this scenario often in development environments, and hostname/IP-based config will probably be necessary on K8s or clusters without external DNS.  The same applies to server config.

Reply:
Good idea, we can answer a lot of potential questions for this. I can add some details for this.